client/lib/domains/dns/index.js

```diff
@@ -61,8 +61,7 @@ function isValidName( name, type, selectedDomainName ) {
 	switch ( type ) {
 		case 'CNAME':
```

Comment:
Given that this case and the `SRV` one are now the same we probably don't need a switch statement here anymore.